x-pack/plugins/snapshot_restore/public/app/components/repository_form/_repository_form.scss

```diff
@@ -0,0 +1,19 @@
+@import 'src/legacy/ui/public/styles/_styling_constants';
+
+.ssrRepositoryFormTypeCard {
```

Comment:
@elastic/kibana-design the rules here are to add hover and selected styling to the repository type cards. there is also a wrapper around each card to enable keyboard navigation and selection. tabbing through each card applies the same styling as hovering:

![Apr-25-2019 08-58-00](https://user-images.githubusercontent.com/1965714/56750295-50075c80-6738-11e9-8f8d-e79cd7c1a9e5.gif)

Reply:
We are working on an EUI-side solution for you, can you hold off on applying this?

Reply:
@cchaos I'm happy to swap out with EUI's solution whenever it is ready and would like to keep current styling in place until then 😄